docroot/modules/custom/sitenow_seo/sitenow_seo.module

```diff
@@ -22,7 +22,7 @@ function sitenow_seo_form_google_analytics_admin_settings_alter(&$form, FormStat
 /**
  * Implements hook_metatags_alter().
  */
-function sitenow_metatags_alter(array &$metatags, array &$context) {
+function sitenow_seo_metatags_alter(array &$metatags, array &$context) {
```

Comment:
Wow where these hooks not working then?

Reply:
i couldn't figure out a way to test this one.

Reply:
the other was just a couple days live. i moved it from the profile and merged it but copied the function name from the one above.

Reply:
I must've goofed this up a long time ago. I think I can test it.

Reply:
Where are we using a non-AJAX pager anymore?

Reply:
Original issue https://github.com/uiowa/uiowa/issues/115

Reply:
Oh well. I'm sure its fine. Only needed for non-AJAX paged views but I can't find any.